jscomp/main/rescript_compiler_main.ml

```diff
@@ -200,7 +169,7 @@ let eval (s : string) ~suffix =
   let tmpfile = Filename.temp_file "eval" suffix in 
   Ext_io.write_file tmpfile s;   
   anonymous  ~rev_args:[tmpfile];
-  Ast_reason_pp.clean tmpfile
+  if not !Clflags.verbose then try Sys.remove tmpfile with _ -> ()
```

Comment:
Is this from the deleted file in the line above?

Reply:
Yes